docs/developer.rst

```diff
@@ -65,6 +65,47 @@ To build ``jaxlib`` without CUDA GPU support (CPU only), drop the ``--enable_cud
   python build/build.py
   pip install -e build  # installs jaxlib (includes XLA)
 
+
+Additional Notes for Building ``jaxlib`` from source on Windows
+...............................................................
+
+On Windows, following `Install Visual Studio <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_
+to setup latest C++ toolchain. If you need to build with CUDA enabled, follow
+`CUDA Installation Guide <https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html>`_
+to setup CUDA environment.
+
+Install `bazelisk <https://github.com/bazelbuild/bazelisk/releases/latest>`_.
+Download and rename it as ``bazel.exe`` and add it to your path.
+
+It is recommended to use `Anaconda <https://docs.anaconda.com/anaconda/install/windows/>`_
+or `Miniconda <https://docs.conda.io/en/latest/miniconda.html#windows-installers>`_
+to setup python environment.
+
+Some targets of Bazel use bash utilities to do scripting, so `MSYS2 <https://www.msys2.org>`_
+is needed. See `Installing Bazel on Windows <https://docs.bazel.build/versions/master/install-windows.html#installing-compilers-and-language-runtimes>`_
+for more details. Install the following packages::
+
+  pacman -S patch realpath
+
+
+Once everything is installed. Open PowerShell, make sure you've added MSYS2
+path to current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
+accessible. Activate conda environment. Folllowing command build with cuda
```

Comment:
Activate the conda environment. The following command builds with CUDA enabled::